### PR TITLE
Extend RTL functionality in desktop ✨

### PR DESCRIPTION
**Pull request by Enriqe**
Part of #11647
Extending support for RTL in `amp-story`. 

In the desktop screen:
* Flips the navigational buttons. 
* Places the next page to the left of the active page.
* Places the previous page to the right of the active page.

More follow-ups to come.

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story-desktop.css [hunk 1]
```diff
@@ -132,15 +132,32 @@ amp-story[standalone][desktop] {
   background: linear-gradient(to right, rgba(33,33,33,0) 0%, rgba(33,33,33,0.32) 100%)!important;
 }
 
+[dir=rtl] [desktop] > .next-container::before {
+  right: auto !important;
+  left: 0 !important;
+  background: linear-gradient(to right, rgba(33,33,33,0.32) 0%, rgba(33,33,33,0) 100%)!important;
+}
+
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-desktop.css]: For all these new `rtl` rules, can we reuse the existing CSS and simply add new selectors? For example, I think these properties apply to both:
- `prev-container` if `ltr`
- `next-container` if `rtl`

Which could reduce code duplication, and give something like:

````
[desktop] > .prev-container::before,
[dir=rtl] [desktop] > .next-container::before {
  left: 0 !important;
  right: auto !important;
  background: linear-gradient(to right, rgba(33,33,33,0.32) 0%, rgba(33,33,33,0) 100%)!important;
}
````

File: extensions/amp-story/1.0/amp-story-desktop.css [hunk 2]
```diff
@@ -248,17 +232,36 @@ dragons here. */
...
+  right: auto !important;
 }
 
-.i-amphtml-story-fwd-next > .i-amphtml-story-button-move {
+/* [dir=rtl] .prev-container > .i-amphtml-story-button-move {
+  left: auto !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg width="16" height="25" xmlns="http://www.w3.org/2000/svg"><path d="M.3 3l9.5 9.5L.3 22l3 3 12.4-12.5L3.2 0"/></svg>')!important;
+  background-position: 55% 50%!important;
+  right: 0!important;
+} */
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-desktop.css]: Please remove

File: extensions/amp-story/1.0/amp-story-desktop.css [hunk 3]
```diff
@@ -248,17 +232,36 @@ dragons here. */
...
+  left: auto !important;
+  right: 0 !important;
 }
 
+/* [dir=rtl] .i-amphtml-story-fwd-next > .i-amphtml-story-button-move {
+  right: auto !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg width="16" height="25" xmlns="http://www.w3.org/2000/svg"><path d="M15.7 22l-9.5-9.5L15.7 3l-3-3L.4 12.6 12.8 25"/></svg>')!important;
+  background-position: 45% 50%!important;
+  left: 0!important;
+} */
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-desktop.css]: Please remove

**Comment by Enriqe** [comment]
Ready for merge @newmuis

**Pull request merged**